config/locales/en.yml

```diff
@@ -25,11 +25,33 @@ en:
       strip_insignificant_zeros: false
       unit: "$"
 
-  titles:
-    application: Budgetapp
+  application:
+    show:
+      button: "New Account"
+      new_account: "To sign up for a new account click the 'New Account' button"
+      welcome: "Welcome to Budget App"
+
+  users:
+    new:
+      name: "Name"
+      email: "Email"
+      password: "Password"
+      submit: "Sign up"
+      title: "Please fill in the form to sign up"
+    edit:
+      name: "Name"
+      delete: "Delete Account"
+      email: "Email"
+      submit: "Edit profile"
+      title: "Please fill in the form to edit your profile"
+    show:
+      output: "Your email address is: "
```

Comment:
Is `output` the most descriptive key we could use? or is there a better one?

Reply:
Lol, probably not, sorry, was late in the day, thanks for the pick up!